src/styles/index.scss

```diff
@@ -1,3 +1,2 @@
-body {
-  margin: 0;
-}
+@import './utils/variables';
+@import './blocks/card';
```

Comment:
Ensure that you also import the `stars` block SCSS file here, as it should be in a separate file according to the task requirements.